process.go

```diff
@@ -21,7 +17,4 @@ type ProcessConfig struct {
 	Stdin  io.Reader
 	Stdout io.Writer
 	Stderr io.Writer
-
-	// Console is the path to the pty slave for use by the master
-	Console string
```

Comment:
@crosbymichael  @mrunalp  From my point of view, a process config should contain (env, uid, gid, caps, tty, lsm or selinux labels, etc). A container config should contain configuration fror namespaces and cgroups and default parameters for processes.
Now we have a container config with default parameters. In future we will be able to expand the process config, when it's required.